manifest.yaml

```diff
@@ -259,6 +259,10 @@
   BuildScript: ./iOS/SCCatWaitingHUD/build.cake
   TriggerPaths: [ iOS/SCCatWaitingHUD ]
   MacBuildTargets: [ samples, component ]
+- Name: SimplePing
+  BuildScript: ./iOS/SimplePing/build.cake
+  TriggerPaths: [ iOS/SimplePing ]
+  MacBuildTargets: [ component, samples ]
```

Comment:
Should we reverse the target order to match the others?

` MacBuildTargets: [ samples, component ]`